nbconvert/nbconvertapp.py

```diff
@@ -378,9 +398,13 @@ def convert_single_notebook(self, notebook_filename):
             4. (Maybe) postprocess the written file
 
         """
-        self.log.info("Converting notebook %s to %s", notebook_filename, self.export_format)
-        resources = self.init_single_notebook_resources(notebook_filename)
-        output, resources = self.export_single_notebook(notebook_filename, resources)
+        if self.from_stdin:
+            self.log.info("Converting notebook from stdin to stdout", shell_input, self.export_format)
```

Comment:
the `shell_input` here is not needed, and the `self.export_format` here is not used, you might want something like:

`"Converting notebook read from stdin to %s on  stdout", self._export_format"` , A native english Speaker, might find a better sentence.